docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deploy to two data centers with asynchronous replication
+headerTitle: Asynchronous Replication
+linkTitle: Asynchronous replication
+description: Enable deployment using asynchronous replication between two data centers.
+menu:
+  latest:
+    parent: create-deployments
+    identifier: async-replication-platform
+    weight: 633
+aliases:
+type: page
+isTocNested: true
+showAsideToc: false
+---
+
+You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+
+If you are using Yugabyte Platform to manage universes, you need to call the following REST API endpoint on your Yugabyte Platform instance for the producer universe and the consumer universe involved in the asynchronous replication between two data sources:
```

Comment:
wait, I'm confused, we're just calling an endpoint on each universe, but not giving it any information about the other universe? who from platform eng has more context on this?

for wording, we probably also need to clarify, that this can only link 2 universes, that are managed by the same platform (assuming that's true)

Reply:
probably best actually from someone from platform to review this whole file

Reply:
I believe this content was reviewed before, so it should be technically correct. It's not new, the two files existed in our doc library as one file that described both DB and Platform. My task was to split it into two separate files, one for db and one for platform. 
But I'll see if I can get somebody to review the platform file.

Reply:
@chirag-yb Could you, please, review docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md to make sure it's correct? (there hasn't been any changes to the technical part of the content; just formatting/structure)

Reply:
@daniel-yb to review content related to API calls.

Reply:
hey @bmatican this is more of a workaround in the short term for getting the universe version reset in YW since the rest of the steps currently happen out of band of YW (hence there being no need to provide any actual payload to the APIs with context for 2dc setup at the moment). We just "purposefully mislabeled" the API to have it fit in nicely with the flow instead of explicitly calling the API something like "reset universe version". This is sort've a short term crutch until setting up 2dc can be natively supported through the platform